test/suites/init_preseed.sh

```diff
@@ -0,0 +1,60 @@
+test_init_preseed() {
+  # - lxd init --preseed
+  LXD_INIT_DIR=$(mktemp -d -p "${TEST_DIR}" XXX)
+  chmod +x "${LXD_INIT_DIR}"
+  spawn_lxd "${LXD_INIT_DIR}" false
+  (
+    set -e
+    # shellcheck disable=SC2034
+    LXD_DIR=${LXD_INIT_DIR}
+
+    cat <<EOF | lxd init --preseed
+config:
+  core.https_address: 127.0.0.1:9999
+  images.auto_update_interval: 15
+pools:
+- name: data
+  driver: dir
+  config:
+    source: ""
```

Comment:
I've populated the config in case the zfs backend is used. It gives slightly broader coverage to the feature.